node/src/server-modules/GlideJson.ts

```diff
@@ -661,4 +661,82 @@ export class GlideJson {
 
         return _executeCommand<ReturnTypeJson<number>>(client, args);
     }
+
+    /**
+     * Reports memory usage in bytes of a JSON object at the specified `path` within the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param value - The value to append to the string. Must be wrapped with single quotes. For example, to append "foo", pass '"foo"'.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) `path`: The path within the JSON document, defaults to root (`"."`) if not provided.
```

Comment:
Please update - this very specific command returns total memory usage if no path given